web/viewer.css

```
@@ -90,6 +90,7 @@
   --secondaryToolbarButton-spreadNone-icon: url(images/secondaryToolbarButton-spreadNone.svg);
   --secondaryToolbarButton-spreadOdd-icon: url(images/secondaryToolbarButton-spreadOdd.svg);
   --secondaryToolbarButton-spreadEven-icon: url(images/secondaryToolbarButton-spreadEven.svg);
+  --secondaryToolbarButton-toggleReadingDirection-icon: url(images/secondaryToolbarButton-toggleReadingDirection.svg);
```

Please change the filenames/references according to:
```suggestion
  --secondaryToolbarButton-readingDirectionToggle-icon: url(images/secondaryToolbarButton-readingDirectionToggle.svg);
```